priv/static/Linux.sh

```diff
@@ -177,7 +177,7 @@ function get() {
         echo "$1 is already installed. Skipping..."
     else
         echo "Installing $1..."
-        if [[ $1 == "Homebrew" || $1 == "Erlang" ]]; then
```

Comment:
Homebrew should be capitalized.

Reply:
In the installation message, homebrew is actually all lowercase
```
....
echo "1) Zsh"
echo "2) homebrew"
....
```
Should I do capitalization on that too? or just this one?

Reply:
Wherever it's used.